src/pages/tools/WarshallAlgorithm.tsx

```diff
@@ -136,25 +142,26 @@ const WarshallAlgorithm = () => {
                         {graph.map((row, i) => (
                             <div key={i}>
                                 {row.map((value, j) => (
-                                    i == j ? 
-                                    <input
-                                        className="border rounded-md p-1"
-                                        key={j}
-                                        type="text"
-                                        value={0}
-                                        readOnly={true}
-                                        style={{ width: '50px', margin: '2px', border: '2px solid #aaa' }}
-
-                                    /> : 
-                                    <input
-                                        className="border rounded-md p-1"
-                                        key={j}
-                                        type="text"
-                                        value={value === Infinity ? 'inf' : value}
-                                        onChange={(e) => handleGraphChange(i, j, e.target.value)}
-                                        style={{ width: '50px', margin: '2px', border: '2px solid black' }}
-                                    />
+                                    i === j ? (
+                                        <input
+                                            className="border rounded-md p-1"
+                                            key={j}
+                                            type="text"
+                                            value={0}
+                                            readOnly={true}
+                                            style={{ width:'50px', margin: '3px', border: '2px solid #aaa' }}
```

Comment:
The border stated here is inconsistent with overall borders applied on this page.